apps/backend/src/location/location.service.ts

```diff
@@ -99,56 +98,22 @@ export class LocationService {
     );
     await this.ministryLocationRepo.save(
       this.ministryLocationRepo.create(
-        Object.values(locationEntityList).map((itm) => itm)
+        Object.values(MasterLocationEntityList).map((itm) => itm)
       )
     );
   }
 
-  /**
-   * Build a normalized location list from a list of locations in order to match the merchant ids to a single location id
-   * @param locations
-   * @returns
-   */
-  public normalizeLocations(locations: LocationEntity[]): NormalizedLocation[] {
-    const normalizedLocationList = locations.reduce(
-      (acc: { [key: string]: NormalizedLocation }, itm: LocationEntity) => {
-        const key = itm.location_id;
-        if (!acc[key]) {
-          acc[key] = {
-            source_id: itm.source_id,
-            location_id: itm.location_id,
-            program_code: itm.program_code,
-            ministry_client: itm.ministry_client,
-            resp_code: itm.resp_code,
-            service_line_code: itm.service_line_code,
-            stob_code: itm.stob_code,
-            project_code: itm.project_code,
-            merchant_ids: [],
-            pt_location_id: 0,
-            description: '',
-          };
-        }
-        itm.merchant_id !== BankMerchantId &&
-          acc[key].merchant_ids.push(itm.merchant_id);
-        if (itm.method === LocationMethod.Bank) {
-          acc[key].pt_location_id = itm.pt_location_id;
-          acc[key].description = itm.description;
-        }
-        return acc;
-      },
-      {}
-    );
-    return Object.values(normalizedLocationList);
-  }
-  public async createLocations(locationsData: LocationEntity[]): Promise<void> {
+  public async createLocations(
+    locationsData: MasterLocationEntity[]
+  ): Promise<void> {
     await this.locationRepo.save(this.locationRepo.create(locationsData));
   }
 
   public async getLocationsByID(
     program: Ministries,
     location_ids: number[]
-  ): Promise<NormalizedLocation[]> {
-    const locations = await this.locationRepo.find({
+  ): Promise<MinistryLocationEntity[]> {
+    return await this.ministryLocationRepo.find({
```

Comment:
If you're returning straight, I'm pretty sure you don't need the await, as `.find` is already an async function whose results will be properly returned